Hazel/FewerThreads/ThreadLimitedUdpConnectionListener.cs

```diff
@@ -365,7 +367,14 @@ protected virtual void QueueRawData(ByteSpan span, IPEndPoint remoteEndPoint)
         /// <param name="endPoint">Connection key of the virtual connection.</param>
         internal bool RemoveConnectionTo(ConnectionId connectionId)
         {
-            return this.allConnections.TryRemove(connectionId, out var conn);
+            return this.allConnections.TryRemove(connectionId, out _);
```

Comment:
I tried to put the remove here, but this is *before* the server sends its disconnect message and the peer data is required to send anything. So I created a later step in Dispose.